spec/cypress/support/step_definitions/common-interaction.js

```diff
@@ -0,0 +1,40 @@
+import { When, Then } from "cypress-cucumber-preprocessor/steps";
+
+const elements = {
+  "cookie consent": '[name="cookies_form[analytics_consent]"]',
+  "cookie banner": ".js-cookie-banner",
+};
+
+When("I set {string} radio to {string}", (element, value) => {
```

Comment:
Hmm, Do Given/When/Then are interchangeable ? for example could I use the same string pattern and don't care if I use Given/When or Then

Reply:
There doesn't seem to be a way to do that - but given that they all have specific meanings I'm also not sure why we'd want to?

Reply:
It's because a Given for one statement  could be an And for another and yet When for another - in Ruby it doesn't matter whatever statement you use as long as it matches the string